src/WPF/ArcGISRuntime.WPF.Viewer/Samples/MapView/FeatureLayerTimeOffset/readme.md

```diff
@@ -1,5 +1,34 @@
 # Feature layer time offset
 
-Show data from the same service side-by-side with a time offset. This allows for the comparison of data over time.
+Display a time-enabled feature layer with a time offset.
 
-<img src="FeatureLayerTimeOffset.jpg" width="350"/>
+![Image of feature layer time offset](FeatureLayerTimeOffset.jpg)
+
+## Use case
+
+You can use a time offset to compare time periods by displaying them overlaid on the map. For example, you could show a feature layer with flu cases from December overlaid with flu cases from January to visualize the spread of the disease over time.
```

Comment:
When the data is available in a few weeks, it might be interesting to compare the coronavirus spread in Italy with the spread in the US - there's a clear pattern of exponential growth starting about 10 days later in the US, so might resonate more than flu.